site/en/education/recommendations-for-web-views.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Recommendations on using Web Views
+metadesc: Recommendations for developers to stay compliant with content filtering requirements that schools may have when using embedded webviews.
+date: 2022-04-19
+weight: -3
+tags:
+  - web
+---
+
+Education institutions use content filtering products to prevent their users from accessing inappropriate content on their Chromebooks. There are a handful of ways that their users are able to escape this content sandbox and access content that they should otherwise not be able to access. We have compiled a list of recommendations for Android and Chrome App developers targeting the education market that should help with making sure that your applications work well for education users.
+
+## Understanding WebViews in school scenarios
+
+If your app uses WebViews to show content such as to allow students to login to your service or to display documentation, it may enable students to browse the web without the content filtering protections provided by their schools. Some of the most popular content filtering products used by schools on Chromebooks are deployed as Chrome extensions, which can only see traffic within browser tabs. Since web content is often unpredictable, students can often traverse links within the WebView to browse the web freely, without content filtering protections. Carefully evaluate the use of WebViews in your app and consider the alternatives below.
```

Comment:
```suggestion
If your app uses WebViews to allow students to login to your service or to display documentation, it may enable students to browse the web without the content filtering protections provided by their schools. Some of the most popular content filtering products used by schools on Chromebooks are deployed as Chrome extensions, which can only see traffic within browser tabs. Since web content is often unpredictable, students can often traverse links within the WebView to browse the web freely, without content filtering protections. Carefully evaluate the use of WebViews in your app and consider the alternatives below.
```